dencam/buttons.py

```diff
@@ -30,13 +30,13 @@ class ButtonHandler(Thread):
 
     """
 
-    def __init__(self, recorder, state, stop_flag):
+    def __init__(self, recorder, state, Airplanemode, stop_flag):
         super().__init__()
 
         self.recorder = recorder
         self.state = state
         self.stop_flag = stop_flag
-
+        self. Airplanemode = Airplanemode
```

Comment:
this member variable also should follow variable naming rules so self.airplane_mode (or self.airplanemode but I think adding the underscore is better).  There is also a space between the . and the variable name which shouldn't be there. My guess is this should have generated an error when running the code. Did this not crop up when you tested the code?